addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -322,7 +326,9 @@ class ChatWindow extends Component {
         if (!this.chatWindow.thread) {
             return;
         }
-        this._saveThreadScrollTop();
+        if (this._threadRef.comp) {
```

Comment:
There is a preemptive return on thread just above that should be enough to cover this case too no?

Reply:
or with "not folded" added to the condition too

Reply:
Seems to be a different management of isFolded whether there is a thread linked to chat window or not (see _isFolded and isFolded in chat_window model) so not sure we can rely on `chatWindow.isFolded`

Reply:
Well if we think what is the business condition for the thread viewer to be currently shown on the chat window:
- must have thread
- must be open ("unfolded")
- must be visible/not hidden (always the case in this case, hidden menu is instantiating just the header not the full chat window)

So both first conditions should be enough, and those are the conditions that are used in the template to reach the case where ThreadViewer is instanciated.